packages/ses/scripts/hermes-transforms.js

```diff
@@ -75,7 +77,7 @@ export const hermesTransforms = {
     const transforms = {
       ...asyncArrowEliminator,
       ...asyncGeneratorDestroyer,
-      ...removeImmutableArrayBufferShim,
+      ...immutableArrayBufferPonyfier,
```

Comment:
we're not able to replace `import '@endo/immutable-arraybuffer/shim.js';` in ses/lockdown.js with e.g.

```js
const ab = new ArrayBuffer(0);
if (typeof ab.transferToImmutable === 'function') {
  import('@endo/immutable-arraybuffer/shim.js');
} else {
  import('@endo/immutable-arraybuffer/shim-hermes.js');
}
```

so this transform is needed

Reply:
> module magic that lets us conditionally synchronously import one of the other depending on platforms? Are we doing something similar to adapt to XS?

happy to follow-up and replace this transform with it if we do ^ or if we're doing something similar with XS